redactionMap.js

```diff
@@ -57,7 +57,74 @@ const redactionMap = {
         childElementAttribute: 'src',
         value: '/favicon.ico'
       }
-    }
+    },
+    {
+      description: 'See connections (__+), this appears on profile pages top right corner',
+      type: 'class',
+      selector: 'lazy-image profile-picture loaded',
+      replacement: {
+        type: 'attribute',
+        attribute: 'src',
+        value: '/favicon.ico'
+      }
+    },
+    {
+      description: 'Things the person liked (profile page)',
+      type: 'class',
+      selector: 'pv-recent-activity-item__message',
+      replacement: {
+        type: 'text',
+        value: '[Name] liked'
+      }
+    },
+    // This is too general, it hides other important page elements
```

Comment:
Fair enough, go ahead and remove the code entirely rather than commenting it out.